src/components/MovieHeader.ts

```diff
@@ -0,0 +1,29 @@
+import { Logo } from "../../images";
+import { $ } from "../utils/dom";
+
+class MovieHeader extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    this.render();
+    this.addEvent();
+  }
+
+  render() {
+    this.innerHTML = /* html */ `
+    <header>
+    <h1><img src="${Logo}" alt="MovieList" /></h1>
+    <search-box class="search-box"></search-box>
+  </header>`;
+  }
+
+  addEvent() {
+    $("h1", this)?.addEventListener("click", () => {
+      location.reload();
+    });
```

Comment:
콜백에서 호출 인자가 일치하거나 추가적으로 주입해야하는 케이스가 없으면 생략해도 무방합니다!

```suggestion
    $("h1", this)?.addEventListener("click", location.reload);
```

이건 제 느낌인데, `$` 함수에 this를 계속 인자로 넣는 것은 좀 불편해 보이네요...

Reply:
this를 인자로 넣지 않도록 수정하였습니다! 여기서 생략해봤는데 'Illegal Invocation' 에러가 나네요 ㅠㅠㅠ 아마도 또 this 문제인 거 같은데 차차 해결해보겠습니다!